academic_helper/views/basic.py

```diff
@@ -29,6 +29,7 @@ def get_context_data(self, **kwargs):
         log.info(f"Rendering {wrap(self.title)} for {wrap(self.user)}")
         context = super().get_context_data(**kwargs)
         context["title"] = self.title
+        context["messages"]=["I <3 Snacks"]
```

Comment:
If I understand correctly, this will pop up on every page load?

Reply:
yup, that's indeed what's hapenning.